config/environments/development.rb

```diff
@@ -58,4 +58,9 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = { :address => 'localhost', :port => 1025 }
+  config.action_mailer.default_url_options = { :host => "localhost:3000" }
```

Comment:
Was this the magic line that let you use `dashboard_url`?

Reply:
Yup, that's the one; had to do the same thing in `config/environments/test.rb` to get the `url` helper to work in tests